app/controllers/suggestions_controller.rb

```diff
@@ -1,5 +1,7 @@
 class SuggestionsController < ApplicationController
   before_action :load_suggestion, only: %i(show destroy)
+  before_action :authenticate_user!
+  authorize_resource
```

Comment:
sao không dùng load and authorize resource

Reply:
bởi vì `load and authorize resource` gồm 2 hành động `load_resource` và `authorize resource` mà ở đây resource đã được load rồi nên chỉ cần dùng `authorize resource` cũng dc